com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/Templates/VertexAnimation.template.hlsl

```diff
@@ -39,12 +39,16 @@ VertexDescriptionInputs AttributesMeshToVertexDescriptionInputs(AttributesMesh i
     return output;
 }
 
-// This is used for injecting the define below.
-$splice(CustomInterpolatorPreVertex)
+    // This is used for injecting the define below.
+    $splice(CustomInterpolatorPreVertex)
 
 
 
-    AttributesMesh ApplyMeshModification(AttributesMesh input, float3 timeParameters
+    AttributesMesh ApplyMeshModification(AttributesMesh input,
```

Comment:
Invocation of this call is getting pretty messy with both VFX and custom interpolants trying to get their info in

Reply:
unless we can generalise VFX as just a list of interpolant and merge it with regualr custom interpolator feature I guess we have no choice?

Reply:
Indeed we do not have a choice unless we can merge it with custom interpolator. Somewhat related, I have discussed with @PaulDemeulenaere about generalizing a lot of the content of this PR (for example 90% of what is in HDRP package for VFX generation can be moved into the VFX package and shared by all render pipelines). But it will be a sizeable amount of work and we are leaning on doing a later PR to generalize the code.